RocketChat/Rocket.Chat
# [IMPROVE] Rewrite Enter Encryption Password Modal

## Proposed changes (including videos or screenshots)


### before
![image](https://user-images.githubusercontent.com/27704687/123182889-bbf3c580-d466-11eb-8d4d-9cfc3d224e33.png)

### after
![image](https://user-images.githubusercontent.com/27704687/123182916-cada7800-d466-11eb-96ee-850be190d419.png)

### Aditional Improves:
- Added a visual validation in the password field



## Issue(s)


## Steps to test or reproduce

- Make sure to have e2e activated
- Login in to your account on a device for the first time (use the incognito mode is enough)
- Click on the blue banner

## Further comments
